src/labels_by_user_input.py

```diff
@@ -35,3 +45,24 @@ def labels_by_user_input(event_data, pull):
         if f"/un{LABEL_APPROVE}".lower() in body:
             remove_label(pull=pull, label=LABEL_APPROVE)
             set_commit_status_pending_no_approve(commit=last_commit)
+
+    semver_command = get_semver_command(body)
+    if semver_command or semver_label_in_pull(pull):
```

Comment:
why `semver_label_in_pull(pull)` is needed?

Reply:
We have three situations on comment.
1. Semver command (Success)
2. No semver command, semver labels exist on pr (Success)
3. No semver command, semver labels do not exist (Pending)

This ensures that every single run, we exit with a success/pending.

Reply:
Please rename `semver_label_in_pull` to `semver_label_in_pull_labels`